spec/tictactoe_spec.rb

```diff
@@ -35,11 +39,37 @@ def selection(board)
     end
   end
 
+  class TicTacToe::FakeBoard
```

Comment:
This fake board looks a lot like the real board. Do we need to duplicate all of this logic in order to provide a meaningful test double for `TicTacToe`? What are some ways that we can make is simpler?

For example, the rotation of tokens is an implementation detail of the real `Board`? Is there a reason it needs to be included in this fake?

Reply:
Not really.  I'll simplify.